packages/remark-lint-no-paragraph-content-indent/index.js

```diff
@@ -104,7 +104,7 @@ function noParagraphContentIndent(tree, file) {
         return;
       }
 
-      if (first === true && ws(toString(node).charAt(0))) {
+      if (first && ws(toString(node).charAt(0))) {
```

Comment:
why remove `=== true` here...

Reply:
it's a redundant comparison, or do you prefer it for readability?

Reply:
I’m fine with either, but it was a bit inconsistent with the `alt` part!